packages/App/src/Navigation.tsx

```diff
@@ -1,51 +1,137 @@
 import React from 'react';
 import {NavigationContainer} from '@react-navigation/native';
 import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
 import LogIn from './Registaration/LogIn';
 import SignUp from './Registaration/SignUp';
 import Loading from './components/Loading';
 import {userIsLogIn, IsLoadingUser} from './store/selectors/registration';
-import {useSelector} from 'react-redux';
-
-import BalanceNavigation from './balance/BalanceNavigation';
+import {useDispatch, useSelector} from 'react-redux';
+import {StyleSheet, View, Image, Text, Alert} from 'react-native';
+import BalanceNavigation from './balance/wallet/WalletNavigation';
+import DebitNavigation from './balance/debits/DebitNavigation';
+import walletImgSource from '../Pics/TabMenu/creditCard.png';
+import debitsImgSource from '../Pics/TabMenu/money.png';
+import {getError} from './store/selectors/walletItems';
+import {getErrorDebits} from './store/selectors/debits';
+import {logOutAction} from './store/actions/registration';
 
 const Stack = createNativeStackNavigator();
+const Tab = createBottomTabNavigator();
 
 const Navigation = () => {
+  const dispatch = useDispatch();
   const isLogIn = useSelector(userIsLogIn);
   const isLoading = useSelector(IsLoadingUser);
+  const getErrorInfo = useSelector(getError);
+  const debitsError = useSelector(getErrorDebits);
+  console.log(isLogIn);
+
+  if (getErrorInfo || debitsError) {
+    Alert.alert(getErrorInfo || debitsError);
+    dispatch(logOutAction());
```

Comment:
Looks like the user is going to be logged out if the network is unstable or if 404 is returned.

This is incorrect, but it's also a bad architecture pattern to mess UI logic and redux behaviour. Your components should only know how to show errors, not how to handle them. The logOut should happen in sagas and only after you could not re-authenticate user with refresh token.

Reply:
Same for Debits screen